pkg/distributor/distributor.go

```diff
@@ -1099,15 +1093,22 @@ func (d *Distributor) prePushRelabelMiddleware(next PushFunc) PushFunc {
 			return err
 		}
 
+		validationScheme := d.limits.ValidationScheme(userID)
+
 		var removeTsIndexes []int
 		lb := labels.NewBuilder(labels.EmptyLabels())
 		for tsIdx := 0; tsIdx < len(req.Timeseries); tsIdx++ {
 			ts := req.Timeseries[tsIdx]
 
-			if mrc := d.limits.MetricRelabelConfigs(userID); len(mrc) > 0 {
+			if mrcs := d.limits.MetricRelabelConfigs(userID); len(mrcs) > 0 {
```

Comment:
nit to name `mrcs` something intelligible like `relabelConfigs`

Reply:
@juliusmh Can you address the nitpick?